src/common/utils/api.ts

```diff
@@ -43,6 +43,11 @@ const performRequest = async (query: string, parameters: IQueryObject = {}, opti
   if (response.status === 204) {
     return null;
   }
+  // encountered a http error, throw error
+  if (response.status > 399) {
+    throw response;
+  }
```

Comment:
Are we sure this doesn't break any existing code?

Reply:
Changed it to only throw the response when response.json fails and when a 403 or 5xx error happens. This should not break any existing apis. Throwing a error on 403 and 5xx is required by the oidc api in order to show the user an error when their token has expired(403) and when data-dump and anonymize fails due to internal server error.